src/sled_store.rs

```diff
@@ -36,17 +34,8 @@ pub enum SetError {
 }
 
 impl SledStore {
-    pub(crate) fn new(config: &StoreConfig) -> Self {
-        let dirs = ProjectDirs::from(
-            config.qualifier.as_deref().unwrap_or(""),
-            &config.organization,
-            &config.application,
-        );
-        let parent_dir = match dirs.as_ref() {
-            Some(dirs) => dirs.data_dir(),
-            None => Path::new("."), // todo: maybe warn?
-        };
-        let db_path = parent_dir.join("bevy_pkv.sled");
+    pub(crate) fn new(constructor_bundle: Location) -> Self {
```

Comment:
Ditto

Reply:
Where are you getting this old code from?

Reply:
Please fix the name of the argument

Reply:
Sorry, I thought you where asking to fix the `type` of the argument, which had already been done. I'll fix it